C_routing.md

```diff
@@ -649,17 +628,17 @@ Older versions of Phoenix defined a third pipeline `:before`. Its purpose was to
 
 Phoenix defines two other pipelines by default, `:browser` and `:api`. The router will invoke these after it matches a route, assuming we have called `pipe_through/1` with them in the enclosing scope.
 
-As their names suggest, the `:browser` pipeline prepares for routes which render HTML for a browser. The `:api` pipeline prepares for routes which produce data for an api.
+As their names suggest, the `:browser` pipeline prepares for routes which render HTML for a browser. The `:api` pipeline prepares for routes which produce data for an api, for example JSON.
```

Comment:
I don't think that the ", for example JSON" actually adds anything here. Please remove it.